dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.idl

```diff
@@ -113,4 +113,11 @@ namespace Microsoft.Windows.ApplicationModel.Resources
         static String TargetSize { get; };
         static String Theme { get; };
     }
-} // namespace Microsoft.Windows.ApplicationModel.Resources
+
+    [contract(MrtCoreContract, 1)]
+    runtimeclass ApplicationLanguages
+    {
+        static String PrimaryLanguageOverride;
```

Comment:
Has Xaml/WinUI3 been hooked up to use this new API? Or does it all just work?

Reply:
I have tested both manually loading resources with ResourceLoader and XAML (by setting x:Uid property and adding appropriate string resource) and it just works.  Any additional tests I could do here?